elastic/beats
# Update dependencies for M1 support in System

## What does this PR do?

This is the result of updating `go-sysinfo` and `gosigar` and running `go mod tidy`. I'm still in the process of testing Metricbeat as a whole against M1, and some holes in the test tooling aren't helping, so I'm just putting in this PR now.

## Why is it important?

The updates in the upstream system package are needed for M1 Mac support.

## Checklist



## How to test this PR locally

- Pull down PR
- Build metricbeat for darwin/arm
- Make sure it builds.

## Related issues

- https://github.com/elastic/beats/issues/21855
